pkg/generate/code/compare.go

```diff
@@ -364,6 +366,7 @@ func compareMap(
 		// TODO(jaypipes): Implement this by walking the keys and struct values
 		// and comparing each struct individually, building up the fieldPath
 		// appropriately...
+		return ""
```

Comment:
Mmmmm maybe it's better to put a `fallthrough or `panic` statement here instead of returning empty strings

Reply:
> Mmmmm maybe it's better to put a `fallthrough or `panic` statement here instead of returning empty strings

yes, @A-Hilaly, probably best to panic.